portfolio/src/main/webapp/ExtraInfo.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>About Me</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+  <body>
+    <div id="content">
+      <a href = "/index.html"><img src="/images/Home_Button.png"/></a>
+      <p><a href = "/index.html">HOME</a></p>
+      <p><a href = "/AboutMe.html">ABOUT ME</a></p>
+      <p><a href = "/experience.html">EXPERIENCE</a></p>
+      <p><a href = "/ExtraInfo.html">EXTRA INFO</a></p>
+      <p><a href = "/ContactMe.html">CONTACT ME</a></p>
+      <img src="/images/Book_Window.jpg"/>
+      <p>🎓 Expected Graduation Date: May 2024</p>
+      <p>🎯Skills: HTML, Python, C++, CSS, JavaScript, Java, and Figma</p>
```

Comment:
Fine to use a paragraph here but I want to call your attention to `<ul>`, which is for unordered lists. Looks something like this:

```
<ul>
  <li>Item 1</li>
  <li>Item 2</li>
</ul>

Reply:
Thank you! I changed the list around to be an unordered list instead.